Team6220_PowerPlay/src/main/java/org/firstinspires/ftc/team6220_PowerPlay/BaseTeleOp.java

```diff
@@ -217,6 +219,129 @@ public void driveSlidesWithController() {
         }
     }
 
+    public void driveSlidesWithControllerLimitSwitch() {
+        // joystick control
+        slideTargetPosition += (int) (-gamepad2.left_stick_y * 25);
```

Comment:
This was copy-pasted from driveSlidesWithController, so the same can be said about that method (fixing magic numbers wasn't my intention with this branch)

Reply:
It can be changed in a future branch though

Reply:
It's really better to fix things when you see them because usually the future branch never happens

Reply:
Fixed. I don't think that adding the slide height methods in this PR is a good idea as then this commit will have two different untested changes, either of which could cause problems and so I think we should test the slide height methods separately and then create another PR

Reply:
I disagree. It's not really an untested change since it's just moving the switch statement into a method. It'll still work in a different spot. If you can actually promise that you'll do it in another PR and that's easier for you that's fine, but I expect you to do it immediately after this one.

Reply:
Fixed